### ISSUE TITLE
Issue with backslash in php 5.3.10

### ISSUE THREAD

**Issue by KupGest** [slimphp/Slim#666]
Hi guys, 
i've had a problem when deploying to a client server that has a php version 5.3.10

and for no reason when i used the urlFor, it wasn't working (it was concatenating the urls)

so i had to do a little hack to the function in the request.php file

public function __construct(\Slim\Environment $env)
{
    /\* hack _/
    $temp = explode('/', $env['SCRIPT_NAME']);
    if(!empty($temp[0])) $env['SCRIPT_NAME'] = '/'.$env['SCRIPT_NAME'];
    /_ /hack */

```
$this->env = $env;
$this->headers = new \Slim\Http\Headers(\Slim\Http\Headers::extract($env));
$this->cookies = new \Slim\Helper\Set(\Slim\Http\Util::parseCookieHeader($env['HTTP_COOKIE']));
```

}

i don't really know why this was happening, just hope this helps

**Comment by silentworks** [comment]
This seem like the same as #663 and #667 should fix this. If not can you give more details as to the OS the client server is running and version of Slim you are using.

**Comment by codeguy** [comment]
Should be fixed in 2.3.5. If not, let me know.
